pkg/store/copr/coprocessor.go

```diff
@@ -850,9 +858,15 @@ func (worker *copIteratorWorker) run(ctx context.Context) {
 }
 
 // open starts workers and sender goroutines.
-func (it *copIterator) open(ctx context.Context, enabledRateLimitAction, enableCollectExecutionInfo bool) {
+func (it *copIterator) open(ctx context.Context, tryCopLiteWorker *uint32) {
+	if len(it.tasks) == 1 && tryCopLiteWorker != nil && atomic.CompareAndSwapUint32(tryCopLiteWorker, 0, 1) {
```

Comment:
How about using the `boolean` type for `tryCopLiteWorker` if its value is just on or off instead of checking `pointer == nil`?

Reply:
I don't understand, CAS operations need to be done here, `boolean`  can't implement?